cmd/skopeo/signing.go

```diff
@@ -18,6 +18,7 @@ type standaloneSignOptions struct {
 func standaloneSignCmd() cli.Command {
 	opts := standaloneSignOptions{}
 	return cli.Command{
+		Before:    needsRexec,
```

Comment:
Oops, sorry for missing this - `standalone`* should not need reexec ; OTOH `inspect` and `layers` do need it.

Reply:
fixed now